README.md

```diff
@@ -1,30 +1,80 @@
-# Brzeczyk
-A compiler implemented by TCS UJ students at "Compilers" course in the 2022/23 winter semester.
+# Brzęczyk
 
-## Building
-Project can be built using `gradle`. Installation of `gradle` is not required. To build, you can run
-```bash
-./gradlew build
-```
+[ˈbʒɛ̃n͇ʧ̑ɨk]
 
-## Testing
-Tests can be run with
-```bash
-./test_all.sh
-```
+A simple programming language with very *polished* syntax.
+
+Implemented by students at Theoretical Computer Science Department of Jagiellonian University
+as the project for *Compilers* course in the 2022/23 winter semester.
+
+The language does not really have anything out of ordinary and it's mostly C-like, with a few somewhat interesting (compared to C) features:
+
+- nested functions able to access outer variables,
+- dynamically allocated arrays with automatic reference counting,
+- generator functions, allowing to pause the execution and pass a value to the caller.
+
+The compiler is written from scratch and produces x86_64 assembly code.
+
+## Example program
 
-## Formatting
-Code *must* be formatted with `ktlint`. CI will ensure this. You can format your code with
-```bash
-./gradlew ktlintFormat
 ```
+czynność f(x: Liczba) -> Liczba {
+    jeśli (x % 2 == 0)
+        zwróć x / 2
+    wpp
+        zwróć 3 * x + 1
+}
+
+czynność główna() {
+    zm x: Liczba = wczytaj()
+    zm k: Liczba = 0
 
-## Linting
-Linter for this repo is `ktlint`. To view issues with your code you can run:
-```bash
-./gradlew ktlint
+    dopóki (x != 1) {
+        x = f(x)
+        k = k + 1
+    }
+
+    powiedz(„Liczba kroków:”)
+    napisz(k)
+}
 ```
-Any issues with the code will be treated as errors by CI.
 
-## Commits
-Use [imperative commit message convention](https://cbea.ms/git-commit).
+## What?
+
+The following table lists all of the language's keywords and their corresponding meanings.
```

Comment:
Maybe something more informal?
```suggestion
Brzęczyk's keywords can be correctly pronounced by approximately 0.6% of the world's population.
```

Reply:
The formal tone is on purpose.

Reply:
Then please adjust the header name to be formal too